core-bundle/src/Twig/Loader/ContaoFilesystemLoader.php

```diff
@@ -251,6 +252,10 @@ public function exists($name): bool
             return true;
         }
 
+        if ($name instanceof TemplateReference) {
```

Comment:
The current version of the `LoaderInterface` is strictly typed with `string $name` ([ref](https://github.com/twigphp/Twig/blob/3.x/src/Loader/LoaderInterface.php#L48)).

Could you find out your used Twig version?

Reply:
v2.14.6

Reply:
IMHO `string` is the correct type even there:
https://github.com/twigphp/Twig/blob/27e5cf2b05e3744accf39d4c68a3235d9966d260/src/Loader/LoaderInterface.php#L62